tasks/version_powershell.ps1

```diff
@@ -1,14 +1,16 @@
-if (Test-Path 'HKLM:\SOFTWARE\Puppet Labs\Puppet') {
-  $reg = $(Get-ItemProperty -Path 'HKLM:\SOFTWARE\Puppet Labs\Puppet')
-  if (Test-Path $reg.RememberedInstallDir64) {
+$rootPath = 'HKLM:\SOFTWARE\Puppet Labs\Puppet'
+
+$reg = Get-ItemProperty -Path $rootPath -ErrorAction SilentlyContinue
+if ($null -ne $reg) {
+  if ($null -ne $reg.RememberedInstallDir64) {
```

Comment:
I think we only want to set `$loc` if the path exists, can we also `Test-Path`? If the registry entry is there but the file isn't, I interpret that as the package is not installed (but registry cleanup didn't succeed for some reason).

Reply:
We could add that in Line 12

``` powershell
if ( ($null -ne $loc) -and (Test-Path -Path $loc) ) {
```

Reply:
Are you going to follow up on these changes, or would you like me to take over the PR?

Reply:
Feel free to take over.  Just busy with other tasks first.